README.md

```diff
@@ -1,13 +1,19 @@
 Software for Weak Lensing Deblending Studies
 ============================================
 
+[![ADS](https://img.shields.io/badge/ADS-Sanchez%2C%20Mendoza%20et%20al.-blue)](https://ui.adsabs.harvard.edu/abs/2021JCAP...07..043S/abstract)
+[![arXiv](https://img.shields.io/badge/astro--ph.IM-2103.02078-red)](https://arxiv.org/abs/2103.02078)
+[![Documentation Status](https://readthedocs.org/projects/weaklensingdeblending/badge/?version=latest)](https://readthedocs.org/projects/weaklensingdeblending/?badge=latest)
+[![DOI](https://zenodo.org/badge/8727241.svg)](https://zenodo.org/badge/latestdoi/8727241)
+
+
 Weak lensing fast simulations and analysis for the [LSST](http://www.lsst.org/lsst/) [Dark Energy Science Collaboration](http://www.lsst-desc.org/).
 
 This code was primarily developed to study the effects of overlapping sources on shear estimation,
 photometric redshift algorithms, and deblending algorithms.
 
 User and reference documentation is hosted at http://weaklensingdeblending.readthedocs.io/ and also available as a [single pdf document](https://readthedocs.org/projects/weaklensingdeblending/downloads/pdf/latest/).
 
-[![Documentation Status](https://readthedocs.org/projects/weaklensingdeblending/badge/?version=latest)](https://readthedocs.org/projects/weaklensingdeblending/?badge=latest)
-[![DOI](https://zenodo.org/badge/8727241.svg)](https://zenodo.org/badge/latestdoi/8727241)
+If you use this software please cite [2021JCAP...07..043S](https://ui.adsabs.harvard.edu/abs/2021JCAP...07..043S/abstract) and [this zenodo link](https://zenodo.org/record/3975230#.YVczD2ZKjAM).
```

Comment:
Should everyone cite the Zenodo and the JCAP ? or only people that use the code directly should cite the Zenodo?